Spigot-Server-Patches/0582-Add-ServerGUI-image.patch

```diff
@@ -0,0 +1,404 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: AlexProgrammerDE <40795980+AlexProgrammerDE@users.noreply.github.com>
+Date: Fri, 2 Oct 2020 20:42:55 +0200
+Subject: [PATCH] Add ServerGUI image
+
+
+diff --git a/src/main/java/net/minecraft/server/ServerGUI.java b/src/main/java/net/minecraft/server/ServerGUI.java
+index e51434f2323bcb9385d11e99d0438d66434e3e63..3a484f421f1d0eafefedb821bb6e9902cca5f725 100644
+--- a/src/main/java/net/minecraft/server/ServerGUI.java
++++ b/src/main/java/net/minecraft/server/ServerGUI.java
+@@ -10,18 +10,12 @@ import java.awt.event.FocusAdapter;
+ import java.awt.event.FocusEvent;
+ import java.awt.event.WindowAdapter;
+ import java.awt.event.WindowEvent;
++import java.io.IOException;
+ import java.util.Collection;
++import java.util.Objects;
+ import java.util.concurrent.atomic.AtomicBoolean;
+-import javax.swing.JComponent;
+-import javax.swing.JFrame;
+-import javax.swing.JList;
+-import javax.swing.JPanel;
+-import javax.swing.JScrollBar;
+-import javax.swing.JScrollPane;
+-import javax.swing.JTextArea;
+-import javax.swing.JTextField;
+-import javax.swing.SwingUtilities;
+-import javax.swing.UIManager;
++import javax.imageio.ImageIO;
++import javax.swing.*;
+ import javax.swing.border.EtchedBorder;
+ import javax.swing.border.TitledBorder;
+ import javax.swing.text.AttributeSet;
+@@ -32,7 +26,7 @@ import org.apache.logging.log4j.Logger;
+ 
+ public class ServerGUI extends JComponent {
+ 
+-    private static final Font a = new Font("Monospaced", 0, 12);
++    private static final Font a = new Font("Monospaced", Font.PLAIN, 12); // Paper
+     private static final Logger LOGGER = LogManager.getLogger();
+     private final DedicatedServer c;
+     private Thread d;
+@@ -42,26 +36,34 @@ public class ServerGUI extends JComponent {
+     public static ServerGUI a(final DedicatedServer dedicatedserver) {
+         try {
+             UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
+-        } catch (Exception exception) {
+-            ;
++        } catch (Exception ignored) { // Paper
```

Comment:
Nothing changed?

Reply:
I looked kinda messy to me, but if thats bad imma revert it.

Reply:
the constants are useful in case the library decides to change some values and makes the code way more readable since the are a small description of what they are. Just numbers would require someone to manually search for the value of it.